src/app/components/reports-page/reports-page.component.ts

```diff
@@ -20,8 +20,8 @@ import { Filter } from 'src/app/models/Filter';
 export class ReportsPageComponent implements OnInit {
 
   readonly formats: string[] = ["Code", "Document", "Powerpoint", "All"];
-  selFormat: string = "All";
-  selFormatFilter = "All";
+  selFormat: string[] = ["All"];
```

Comment:
Shouldn't the reports page be refactored to use the different checkboxes similarly to the other pages? It seems odd to have an array that has Code, Document, and Powerpoint in addition to "All". It seems that "All" should be removed, and can be checked if the array has the other 3 formats together. Otherwise, we're running into scenarios that the selectedFormats could be ["Code", "All"], which is frankly confusing. If this is necessary, there should be an explanation somewhere as to why.

Reply:
Completed